server/services/notify/plugindelivery/mention_deliver.go

```diff
@@ -24,12 +29,17 @@ func (pd *PluginDelivery) MentionDeliver(mentionUsername string, extract string,
 		}
 	}
 
+	// make sure mentioned user has permissions to team.
+	if !pd.permissions.HasPermissionToTeam(user.Id, evt.TeamID, model.PermissionViewTeam) {
```

Comment:
I'm not sure if this should be the ViewBoard permission, instead of the view team.

Reply:
That would mean you can only mention users who are already members of the board.  Doesn't seem right to me.

Reply:
Yes, I think you are right.